pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -152,6 +152,12 @@ class PubNubImpl(private val pubNubObjC: KMPPubNub) : PubNub {
         )
     )
 
+    companion object {
+        fun create(kmpPubNub: Any): PubNubImpl {
```

Comment:
It seems weird that you can't just call the constructor... this should work I think.
Anyway, this `create` is in iOS only so it's fine, but I'd be curious to see what the error is with the constructor, maybe we can fix it

Reply:
Yeah, it' really weird. I see the original method I want to invoke is present in KMP binary:

![pubnubimpl_init](https://github.com/user-attachments/assets/6aabcf32-5f38-42ba-8b54-9bbab6e4a6c6)

However, here's the error when I want to invoke it. It says `argument passed to call that takes no arguments`:

![pubnubimpl_error_usage](https://github.com/user-attachments/assets/a337a972-2c95-4dfc-8117-11cb08b584f5)